content/codespaces/getting-started-with-codespaces/getting-started-with-your-python-project-in-codespaces.md

```diff
@@ -0,0 +1,222 @@
+---
+title: Getting started with your Python project in Codespaces
+shortTitle: Getting started with your Python project
+intro: 'You can create a custom dev container with all the tools necessary to get started with your Python project in {% data variables.product.prodname_codespaces %}.'
+product: '{% data reusables.gated-features.codespaces %}'
+versions:
+  free-pro-team: '*'
+topics:
+  - Codespaces
+---
+
+
+### Introduction
+
+This guide shows you how to set up your Python project in {% data variables.product.prodname_codespaces %}. It will take you through an example of opening your project in a codespace, and adding and modifying a dev container configuration from a template.
+
+#### Prerequisites 
+
+- You should have an existing Python project in a repository on {% data variables.product.prodname_dotcom_the_website %}. If you don't have a project, you can try this tutorial with the following example: https://github.com/2percentsilk/python-docs-hello-world.
```

Comment:
I'm open to a more "real" sample if we can find one but we need to ensure it just "works" with the devcontainer definition we add (or works with a very minor update we can capture in this doc).

Reply:
A really simple sample probably doesn't hurt because it gets the point across (how to add a Python dev container) and the user can apply the principles learned here to a more advanced project.

As an alternative, here's the project we used in our Learn module: https://github.com/MicrosoftDocs/mslearn-python-products.

Reply:
We use [this one](https://github.com/MicrosoftDocs/mslearn-python-products) for Remote-Containers tutorials which is a Flask app that just displays a list of products. 

![image](https://user-images.githubusercontent.com/686963/117152593-c9060980-ad7f-11eb-86b7-747ad5f8fd47.png)